target/start-mailserver.sh

```diff
@@ -543,13 +543,25 @@ function _setup_dovecot
 
   # set mail_location according to mailbox format
   case "${DOVECOT_MAILBOX_FORMAT}" in
-    sdbox|mdbox|maildir )
+    sdbox|mdbox )
```

Comment:
Why has maildir been removed here? The cron job is not needed for that, but how about the other changes?

Reply:
maildir is the default behavior